README.md

```diff
@@ -1 +1,39 @@
-# ToDoList
\ No newline at end of file
+# ToDoList
+A To-do-ist javascript app buit using webpack and es6.
+
+Prerequisties
+HTML
+CSS
+JavaScript
+Wepack
+
+Getting Started
+Make sure you have node installed on your local machine.
+
+step 1: Clone this repo locally using git in the command line and the following command
+
+git clone https://github.com/asnganje/ToDoList.git
+
+Step 2: run npm install to download the necessary modules.
+
+step 3: run npm start to load the project in your broswer.
+
+
+Author  👤 Abdulrahman Nganje
+
```

Comment:
[OPTIONAL]

- As the Previous reviewer suggested, I would also recommend making the README professional. If you want to take any reference, you can use the [template given by the microverse](https://github.com/microverseinc/readme-template).